packages/account-v2/src/containers/VerificationStatus/ErrorList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { DerivLightDeclinedPoiIcon } from '@deriv/quill-icons';
+import { IconWithMessage } from '../../components/IconWithMessage';
+import { ErrorListItem } from './ErrorListItem';
+
+export const ErrorList = ({ errorList }: { errorList: string[] }) => (
+    <IconWithMessage
+        icon={<DerivLightDeclinedPoiIcon height={128} />}
+        title='Your proof of identity submission failed because:'
+    >
+        <div>
+            {errorList.length < 2 ? (
```

Comment:
what if the length is 0?

Reply:
That will not happen as the place this is bing used has a check 
```
    if (errorMessageList.length) {
        return <ErrorList errorList={errorMessageList} />;
    }
```